packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -224,7 +224,45 @@ class DefaultTextEditingShortcuts extends Shortcuts {
   //   * Meta + backspace
   static const Map<ShortcutActivator, Intent> _androidShortcuts = _commonShortcuts;
 
-  static const Map<ShortcutActivator, Intent> _fuchsiaShortcuts = _androidShortcuts;
+  // Fuchsia keyboard HID usage values are defined in (page 53):
+  // https://www.usb.org/sites/default/files/documents/hut1_12v2.pdf
+  static const int _kFuchsiaKeyIdPlane = LogicalKeyboardKey.fuchsiaPlane;
+
+  static const LogicalKeyboardKey _kBackspace = LogicalKeyboardKey(42 | _kFuchsiaKeyIdPlane);
+  static const LogicalKeyboardKey _kDelete = LogicalKeyboardKey(76 | _kFuchsiaKeyIdPlane);
+  static const LogicalKeyboardKey _kArrowLeft = LogicalKeyboardKey(80 | _kFuchsiaKeyIdPlane);
+  static const LogicalKeyboardKey _kArrowRight = LogicalKeyboardKey(79 | _kFuchsiaKeyIdPlane);
+  static const LogicalKeyboardKey _kArrowDown = LogicalKeyboardKey(81 | _kFuchsiaKeyIdPlane);
+  static const LogicalKeyboardKey _kArrowUp = LogicalKeyboardKey(82 | _kFuchsiaKeyIdPlane);
+
+  static const Map<ShortcutActivator, Intent> _fuchsiaShortcuts = <ShortcutActivator, Intent>{
```

Comment:
This `Map` only includes text editing related shortcuts that are mapped to different `Intent`s after #90684.